dockerfiles/lint.Dockerfile

```diff
@@ -13,7 +14,11 @@ WORKDIR /src
 
 FROM base
 ENV GOFLAGS="-buildvcs=false"
+ARG TIMEOUT
+ARG BUILDTAGS
+ENV TIMEOUT=${TIMEOUT}
+ENV BUILDTAGS=${BUILDTAGS}
```

Comment:
No need to persist these in an env-var; it's only used in the `RUN`, and not used at runtime (I don't even think we even run the image, only used the build to run the linter)

Reply:
The scope and propagation are kinda sad in Dockerfiles (https://docs.docker.com/build/building/variables/#scoping); I feel like I often need a PhD  to get those right